omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bookshelf_validator.rb

```diff
@@ -67,7 +67,7 @@ def verify_storage_type_unchanged
 
 Please set
 
-bookshelf['storage_type'] = :#{previous_value}
+bookshelf['storage_type'] = '#{previous_value}'
```

Comment:
I'm confused about how this previously worked. IRB doesn't seem to like `:#{previous_value}`, so I'm unsure, but it seems like we want double quote here? We probably want to eval `previous_value` instead of passing `#{previous_value}` to storage_type?